core/arch/arm/plat-hikey/conf.mk

```diff
@@ -1,45 +1,43 @@
 include core/arch/$(ARCH)/plat-$(PLATFORM)/platform_flags.mk
 
-ifeq ($(CFG_ARM64_core),y)
-CFG_WITH_LPAE := y
-else
-CFG_ARM32_core ?= y
-CFG_MMU_V7_TTB ?= y
-endif
-
 core-platform-cppflags	+= -I$(arch-dir)/include
-
 core-platform-subdirs += \
 	$(addprefix $(arch-dir)/, kernel mm tee) $(platform-dir)
 
-libutil_with_isoc := y
-libtomcrypt_with_optimize_size := y
+$(call force,libutil_with_isoc,y)
+$(call force,CFG_GENERIC_BOOT,y)
+$(call force,CFG_HWSUPP_MEM_PERM_PXN,y)
+$(call force,CFG_PL011,y)
+$(call force,CFG_PM_STUBS,y)
+$(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
+$(call force,CFG_WITH_ARM_TRUSTED_FW,y)
 
-CFG_WITH_ARM_TRUSTED_FW := y
-CFG_SECURE_TIME_SOURCE_CNTPCT ?= y
-CFG_PL011 ?= y
-CFG_HWSUPP_MEM_PERM_PXN ?= y
-CFG_WITH_STACK_CANARIES ?= y
-CFG_TEE_CORE_EMBED_INTERNAL_TESTS ?= n
-CFG_GENERIC_BOOT ?= y
-CFG_PM_STUBS ?= y
-CFG_CRYPTO_SHA256_ARM32_CE ?= $(CFG_ARM32_core)
-CFG_CRYPTO_SHA256_ARM64_CE ?= $(CFG_ARM64_core)
+ifeq ($(CFG_ARM64_core),y)
+$(call force,CFG_WITH_LPAE,y)
```

Comment:
Yes, as well as `$(call force,CFG_MMU_V7_TTB,n)` here and `$(call force,CFG_WITH_LPAE,n)` in the `else` branch.